src/components/ModelServerDetails.js

```diff
@@ -13,7 +13,8 @@
 * limitations under the License.
 **/
 import React, { Component } from 'react';
-import { translate } from '../../localization/localize.js';
+import { translate } from '../localization/localize.js';
+import { maskPassword } from '../utils/ModelUtils.js';
```

Comment:
It does not make sense to import this `maskPassword` function and then overwrite it on line 33.

Reply:
ah...function at line 33 was moved the ModelUtils, supposed to be removed. Will remove it here.